src/Raven.Server/ServerWide/Commands/PeriodicBackup/UpdateResponsibleNodeForTaskCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands.PeriodicBackup;
+
+public class UpdateResponsibleNodeForTaskCommand : UpdateValueForDatabaseCommand
+{
+    public ResponsibleNodeInfo ResponsibleNodeInfo;
+
+    public const int CommandVersion = 60_001;
```

Comment:
Just to note that if we go with adding a new command it would be very risky to rollback this version. 
Theoretically it is possible to revert, but practically it might be as risky as schema upgrade.